src/bootstrap.rs

```diff
@@ -87,8 +96,12 @@ extern "system" {
 }
 
 /// Locates the Windows App framework package compatible with the (currently internal)
-/// versioning criteria and loads it into the current process. If multiple packages meet
-/// the criteria, the best candidate is selected.
+/// versioning criteria and loads it into the current process.
+///
+/// On error a dialogue box will be displayed. To not have the dialogue box displayed,
```

Comment:
Nit: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/dialog-box-dialog-dialogue

```suggestion
/// On error a dialog will be displayed. To not have the dialog displayed,
```